modules/nodes-pods-vertical-autoscaler-tuning.adoc

```diff
@@ -0,0 +1,162 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-vertical-autoscaler.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="nodes-pods-vertical-autoscaler-tuning_{context}"]
+= Performance tuning the VPA Operator
+
+As a cluster administrator, you can tune the performance of your Vertical Pod Autoscaler Operator (VPA) to limit the rate at which the VPA makes requests of the Kubernetes API server and to specify the CPU and memory resources for the VPA recommender, updater, and admission controller component pods.
+
+Additionally, you can configure the VPA Operator to monitor only those workloads that are being managed by a VPA custom resource (CR). By default, the VPA Operator monitors every workload in the cluster. This allows the VPA Operator accrue historical data for all workloads, which the Operator can use if a new VPA CR is created for a workload. However, this causes the VPA Operator to use significant CPU and memory, which could cause the Operator to fail, particularly on larger clusters. By configuring the VPA Operator to monitor only workloads with a VPA CR, you can save on CPU and memory resources. One trade-off is that if you have a workload that has been running, and you create a VPA CR to manage that workload, the VPA Operator does not have any historical data for that workload. As a result, the initial recommendations are not as useful as those after the workload had been running for some time.
```

Comment:
Is there a limit for how long or how much historical data is kept? Does it overwrite once it reaches a memory or time limit?

Reply:
@eohartman Good question. The Operator keeps 8 days of historical data for each workload. How does this sound:

> This allows the VPA Operator accrue and store 8 days of historical data for all workloads....

Reply:
*to accrue, and maybe 8 should be spelled out as "eight?" 
But otherwise, I think it sounds great!